Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -363,6 +363,22 @@ genderBannerConfig state =
       }
   in config'
 
+disabilityBannerConfig :: ST.HomeScreenState -> Banner.Config
+disabilityBannerConfig state =
+  let
+    config = Banner.config
+    config' = config
+      {
+        backgroundColor = Color.paleLavender
+      , title = (getString NOW_GET_ASSISTED_RIDES)
+      , titleColor = Color.purple
+      , actionText = (getString UPDATE_PROFILE)
+      , actionTextColor = Color.purple
+      , imageUrl = "ny_ic_accessibility_banner_img,https://assets.juspay.in/beckn/merchantcommon/images/ny_ic_banner_gender_feat.png"
```

Comment:
@MercyQueen Why gender banner image is used for accessibility , Please update
https://assets.juspay.in/beckn/merchantcommon/images/ny_ic_banner_gender_feat.png